content/en/docs/usage/kraft_advanced.md

```diff
@@ -0,0 +1,609 @@
+By default, the `kraft` application uses the name `fs0` for the device to mount the filesystem from.
+To use your application with `kraft`, you must name the shared filesystem "fs0".
+
+```
+$ mkdir fs0
+$ echo 'Hello world!' > fs0/file.txt
+$ kraft run
+[...]
+
+SeaBIOS (version 1.13.0-1ubuntu1.1)
+Booting from ROM...
+Powered by
+o.   .o       _ _               __ _
+Oo   Oo  ___ (_) | __ __  __ _ ' _) :_
+oO   oO ' _ `| | |/ /  _)' _` | |_|  _)
+oOo oOO| | | | |   (| | | (_) |  _) :_
+ OoOoO ._, ._:_:_,\_._,  .__,_:_, \___)
+                     Mimas 0.7.0~1939bce
+Hello world!
+```
+
+### Use `kraft run` to debug you application
+
+If you want to debug the application, you can launch it in a paused state by using the `-P, --paused` flags.
+You can run a GDB server for the guest on a specific port using the `-g, --gdb PORT` flags.
+
+```
+$ kraft run -P -g 1234
+```
+
+To connect the GDB to the created server, you can run
+
+```
+$ gdb --eval-command="target remote :1234" build/application-image.dbg
+[...]
+Reading symbols from build/application-image.dbg
+Remote debugging using :1234
+```
+
+You can set the debug information level from the `Build options --> Debug information level` screen in `kraft menuconfig`.
+For more informations about debugging with Unikraft, please check the [debugging section](docs/develop/debugging).
+
+### Overview of `kraft run` command
+
+```text
+Usage: kraft run [OPTIONS] [ARGS]...
+
+Options:
+  -t, --target TARGET        Name of target architecture/platform.
+  -p, --plat PLAT            Target platform.
+  -m, --arch ARCH            Target architecture.
+  -i, --initrd PATH          Provide an init ramdisk.
+  -B, --background           Run in background.
+  -P, --paused               Run the application in paused state.
+  -g, --gdb PORT             Run a GDB server for the guest at PORT.
+  -d, --dbg                  Use unstriped unikernel
+  -n, --virtio-nic NAME      Attach a NAT-ed virtio-NIC to the guest.
+  -b, --bridge NAME          Attach a NAT-ed virtio-NIC an existing bridge.
+  -V, --interface NAME       Assign host device interface directly as virtio-
+			     NIC to the guest.
+  -D, --dry-run              Perform a dry run.
+  -M, --memory INTEGER       Assign MB memory to the guest.
+  -s, --cpu-sockets INTEGER  Number of guest CPU sockets.
+  -c, --cpu-cores INTEGER    Number of guest cores per socket.
+  -w, --workdir PATH         Specify an alternative directory for the library
+			     (default is cwd).
+  -h, --help                 Show this message and exit.
+```
+
+## `kraft clean`
+
+You can use `kraft clean` to clean the build artifacts of your application.
+You can remove the entire build directory by using the `-p, --proper` flags.
+You can also remove the configuration files by using the `-d, --dist` flags.
+
+### Overview of `kraft clean` command
+
```

Comment:
Add some text here, saying something like "You can get a full list of features by running `kraft clean` with no arguments:"